README.md

```diff
@@ -175,7 +175,8 @@ conda create --name pcgrl
 conda activate pcgrl
 pip install tensorflow==1.15
 pip install stable-baselines==2.9.0
-cd gym_pcgrl
+pip install gym==0.19
```

Comment:
The gym gets installed with the PCGRL environment and it always installs the latest version. Check `setup.py`